lemminx-maven/src/test/java/org/eclipse/lemminx/maven/test/LocalPluginTest.java

```diff
@@ -46,65 +48,105 @@ public void tearDown() throws InterruptedException, ExecutionException {
 	}
-	
+
 	// Diagnostic related tests
 
-		@Test(timeout=30000)
-	 	public void testPluginConfigurationDiagnostics() throws IOException, InterruptedException, ExecutionException, URISyntaxException {
-			DOMDocument document = createDOMDocument("/pom-plugin-configuration-diagnostic.xml", languageService);
-			assertTrue(languageService.doDiagnostics(document, () -> {}, new XMLValidationSettings()).stream().map(Diagnostic::getMessage)
-					.anyMatch(message -> message.contains("Invalid plugin configuration")));
-			assertTrue(languageService.doDiagnostics(document, () -> {}, new XMLValidationSettings()).size() == 2);
-		}
-		
-		@Test
-	 	public void testPluginGoalDiagnostics() throws IOException, InterruptedException, ExecutionException, URISyntaxException {
-			DOMDocument document = createDOMDocument("/pom-plugin-goal-diagnostic.xml", languageService);
-			assertTrue(languageService.doDiagnostics(document, () -> {}, new XMLValidationSettings()).stream().map(Diagnostic::getMessage)
-					.anyMatch(message -> message.contains("Invalid goal for this plugin")));
-			assertTrue(languageService.doDiagnostics(document, () -> {}, new XMLValidationSettings()).size() == 2);
-		}
+	@Test(timeout = 30000)
+	public void testPluginConfigurationDiagnostics()
+			throws IOException, InterruptedException, ExecutionException, URISyntaxException {
+		DOMDocument document = createDOMDocument("/pom-plugin-configuration-diagnostic.xml", languageService);
+		assertTrue(languageService.doDiagnostics(document, () -> {
+		}, new XMLValidationSettings()).stream().map(Diagnostic::getMessage)
+				.anyMatch(message -> message.contains("Invalid plugin configuration")));
+		assertTrue(languageService.doDiagnostics(document, () -> {
+		}, new XMLValidationSettings()).size() == 2);
+	}
+
+	@Test
+	public void testPluginGoalDiagnostics()
+			throws IOException, InterruptedException, ExecutionException, URISyntaxException {
+		DOMDocument document = createDOMDocument("/pom-plugin-goal-diagnostic.xml", languageService);
+		assertTrue(languageService.doDiagnostics(document, () -> {
+		}, new XMLValidationSettings()).stream().map(Diagnostic::getMessage)
+				.anyMatch(message -> message.contains("Invalid goal for this plugin")));
+		List<Diagnostic> diags = languageService.doDiagnostics(document, () -> {
+		}, new XMLValidationSettings());
+		System.out.println(diags);
+		assertTrue(languageService.doDiagnostics(document, () -> {
+		}, new XMLValidationSettings()).size() == 2);
+	}
+
+	// Tests regressions related to
+	// https://github.com/eclipse/lemminx-maven/issues/48
+	@Test
+	public void testPluginDiagnosticsIssue48()
+			throws IOException, InterruptedException, ExecutionException, URISyntaxException {
+		DOMDocument document = createDOMDocument("/pom-plugin-configuration-diagnostic-issue48.xml", languageService);
+		System.out.println(languageService.doDiagnostics(document, () -> {
```

Comment:
Remove (although being able to see the diagnostic is kind of useful... maybe further test suite improvements could collect the diagnostic into the test results)